process-engine/process_engine_meta
# 👷 📦 Postgres tests for Jenkins

**Changes:**

1. Add parallel build Step to Jenkins file that runs the tests against PostgreSQL
2. Update the Jenkinsfile to the new design specs employed by the ProcessEngine Runtime
3. Delete obsolete docker testfile
4. Only send a full testlog to slack, if at least one test has failed

PR: #192

## How can others test the changes?

Watch a build for the Consumer API. 
For example:
https://ci.process-engine.io/blue/organizations/jenkins/process-engine_node-lts%2Fprocess_engine_meta/detail/feature%2Fadd_postgres_tests_for_jenkins/3/pipeline

You'll see that the tests are now performed against Sqlite and PostgreSQL.

## PR-Checklist

Please check the boxes in this list after submitting your PR:
